src/worker.js

```diff
@@ -2,31 +2,51 @@ const { workerData, parentPort } = require("worker_threads");
 const chalk = require("chalk");
 const fs = require("fs").promises;
 const { ESLint } = require("eslint");
-
 const eslint = new ESLint(workerData.options);
-
+const path = require("path");
 const formatterPromise = workerData.formatter
   ? eslint.loadFormatter(workerData.formatter)
   : undefined;
 
+function getPrettyPath(absolutePath) {
+  let relativePath = path.relative(process.cwd(), absolutePath);
+  let containingDir = path.dirname(relativePath);
+
+  if (containingDir && containingDir !== '.') {
+      relativePath = path.join(path.basename(containingDir), path.basename(relativePath));
+  }
+
+  return relativePath;
+}
+
+function logMessage(message) {
+  console.log(
+    chalk.dim(new Date().toLocaleTimeString()),
+    chalk.bold(chalk.cyan("[vite-plugin-eslint]")),
+    message
+  );
+}
+
 parentPort.on("message", (path) => {
   eslint
     .isPathIgnored(path)
     .then(async (ignored) => {
       if (ignored) return;
       const [report] = await eslint.lintFiles(path);
+      if (!report) return;
       if (report.output !== undefined) await fs.writeFile(path, report.output);
       if (report.messages.length === 0) return;
       if (formatterPromise) {
         const formatter = await formatterPromise;
-        console.log(await formatter.format([report]));
+        logMessage(await formatter.format([report]));
       } else {
         report.messages.forEach((m) => {
-          const prettyPath = path.slice(path.indexOf("/src/") + 1);
+          const prettyPath = getPrettyPath(path)
```

Comment:
This was not optimal. But always using something relative to cwd is important to keep cmd+click to file.
```suggestion
          const prettyPath = relative(process.cwd(), path)
```